ksy_style_guide.adoc

```diff
@@ -249,28 +248,36 @@ guesswork.
 
 * For simple non-repeated fields, use a simple singular form —
   i.e. `width`, `header`, `transaction_id`, `file`.
-* For repeated files (i.e. with `repeat: something`), use plular form
+* For a sequence (i.e. with `repeat: something`), use plular form
   — i.e. `files`, `transactions`.
 * Don't be overly verbose: use commonly understood abbreviations
   liberally, if it will improve readablity — i.e. `src_mac` or
   `src_mac_addr` instead of `source_media_access_control_address`
+* Repeated fileds which cannot be packed into a sequence should
+  have `id`s containing a number in the end. Numbers may have a
+  visually-obvious structure, like "the first digit is major, the second
+  one is minor".
```

Comment:
This pretty vague and can be interpreted in multiple ways. Probably an example would help. Right now, it's not clear (1) whether `foo_1` or `foo1` fit this, (2) how one starts the numbering and how it should progress, (3) what is "visually-obvious structure".

Reply:
1 I use ```foo0```
2,3 at KSY author's disposal. There are differrent situations and we want to be flexible.

Reply:
Um, yet again, the whole point of having a standard is *not* to be flexible, but to be rigid and provide no-brains-needed solutions, which are also easy to code into machine checks.

Reply:
>the whole point of having a standard is not to be flexible, but to be rigid and provide no-brains-needed solutions, which are also easy to code into machine checks.

If we wanna some checks, IMHO it's better to extend KS language instead of introducing a convention making the names ugly.